WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -24,18 +24,25 @@ final class WooShippingCreateLabelsViewModel: ObservableObject {
     /// View model for the items to ship.
     @Published private(set) var items: WooShippingItemsViewModel
 
-    // TODO: Update this to a property that refers to the package, when selected
-    /// Whether there is a package selected for the shipping label.
-    /// Temporary property that can be set to `true` to enable features that require a selected package, until package feature is complete.
-    let hasPackage: Bool = false
+    /// Selected package for the shipping label.
+    @Published private(set) var selectedPackage: ShippingLabelPackageSelected? {
```

Comment:
is this property even being set somewhere currently?

Reply:
Not yet, but I added it because eventually:

* We'll need some property that can display the selected package details in `WooShippingCreateLabelView`.
* We'll need the selected package to make the request for the label rates for the Shipping Service section.

I added it at least temporarily (while the details of the package selection requests/responses were being worked out) to move forward with fetching the label rates. I've made a note to come back to it once the packages work is done, but feel free to change it at any time if you want or need!